### ISSUE TITLE
var let const

### ISSUE THREAD

**Issue by milankubin** [trackthis/styleguide#4]
We should make proper use of var, let, const when appropriate.
We now make use of 'var' throughout the code close to 100%, where probably 90% should be 'let' or 'const'.
The exception would be, I'm assuming,  generator functions.

**Comment by finwo** [comment]
Correct, the correct behavior isn't written down in the specification yet. I'll write it in the near future.
